tests/support/assert.sh

```diff
@@ -0,0 +1,80 @@
+function _invalid_assert {
+    echo >&2 'No has test run yet. call "eval_prompt" first.'
+    return 1
+}
+
+function assert_ps1_has       { _invalid_assert; }
+function assert_ps1_not       { _invalid_assert; }
+function assert_ps1_is        { _invalid_assert; }
+function assert_ps1_plain_has { _invalid_assert; }
+function assert_ps1_plain_not { _invalid_assert; }
+function assert_ps1_plain_is  { _invalid_assert; }
+function shift_ps1            { _invalid_assert; }
+
+function eval_prompt
+{
+    if [[ -n "$1" ]]; then
+        echo "...check: $1"
+    fi
+
+    export _LP_PS1 _LP_PS1_PLAIN
+    _lp_set_prompt
+    _LP_PS1="$PS1"
+    _LP_PS1_PLAIN="$(strip_colors <<< "$_LP_PS1")"
```

Comment:
Don't use `_LP_` prefix (and neither `LP_`) for variables in the testsuite but not in liquidprompt.